### PR TITLE
Test harness updates for DSE

### PR DESCRIPTION
**Pull request by kishkaru**

### CODE REVIEW

File: support/ccm.rb [hunk 1]
```diff
@@ -458,12 +459,7 @@ def start(jvm_arg=nil)
         attempts = 1
 
         begin
-          if jvm_arg
-            @ccm.exec('start', '--wait-other-notice', '--wait-for-binary-proto', "--jvm_arg=#{jvm_arg}")
-          else
-            @ccm.exec('start', '--wait-other-notice', '--wait-for-binary-proto')
-          end
-
+          @ccm.exec('start', '--wait-for-binary-proto', jvm_arg ? "--jvm_arg=#{jvm_arg}" : '')
```
Comment (stamhankar999) [inline support/ccm.rb]: What happened to the wait-other-notice arg? Was that intentionally removed here?

Comment (kishkaru) [inline support/ccm.rb]: wait-other-notice is depreciated for clusters.

File: support/ccm.rb [hunk 2]
```diff
@@ -600,7 +592,13 @@ def add_node(name)
 
       i = name.sub('node', '')
 
-      @ccm.exec('add', '-b', "-t 127.0.0.#{i}:9160", "-l 127.0.0.#{i}:7000", "--binary-itf=127.0.0.#{i}:9042", name)
+      if @dse
```
Comment (stamhankar999) [inline support/ccm.rb]: If you construct an array of args and then invoke exec with *args, you can reduce duplication here.

Comment (kishkaru) [inline support/ccm.rb]: Cleaned these up.

**Comment by stamhankar999** [comment]
:+1: feel free to merge

**Pull request merged**